docs/_plugins/include_example.rb

```diff
@@ -54,9 +54,9 @@ def render(context)
         puts(e.backtrace)
         exit 1
       end
-      code = select_lines(code)
+      code = select_lines(code).strip
```

Comment:
If we don't strip, the trailing newline is also rendered as a part of the codes.